libs/application/core/src/lib/slugUtil.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  ApplicationConfigurations,
+  ApplicationTypes,
+} from '../types/ApplicationTypes'
+
+export const getTypeFromSlug = (slug: string) =>
+  Object.entries(ApplicationConfigurations).find(
+    ([_, item]) => item.slug === slug,
+  )?.[0] as ApplicationTypes | undefined
```

Comment:
I had to look at this code for a while before realising why you were using Object.entries not Object.values 😄 

I'm wondering if we should just simplify it to be:
```typescript
for (const [key, value] of Object.entries(ApplicationConfigurations) {
  if (value.slug === slug) {
    return key
  }
}

return undefined
```

So that it is more readable